packages/js/src/helpers/shouldShowWebinarPromoNotification.js

```diff
@@ -0,0 +1,40 @@
+import { select } from "@wordpress/data";
+
+/**
+	 * Checks if the Webinar promotion should be shown in the dashboard.
+	 *
+	 * @param {string} store The store to use.
+	 *
+	 * @returns {boolean} Whether the Webinar promotion should be shown.
+	 */
+const shouldShowWebinarPromoNotificationInDashboard = ( store = "yoast-seo/editor" ) => {
+	const isBlackFridayChecklistPromoActive = select( store ).isPromotionActive( "black_friday_2023_checklist" );
+	const isBlackFridayChecklistAlertDismissed = select( store ).isAlertDismissed( "black-friday-2023-sidebar-checklist" );
```

Comment:
Maybe we need to make the promotions also black-friday to be more consistent with the rest of the system

Reply:
Done 👍